Sources/Display.php

```diff
@@ -676,7 +676,7 @@ function Display()
 			SELECT pc.id_choice, pc.label, pc.votes, COALESCE(lp.id_choice, -1) AS voted_this
 			FROM {db_prefix}poll_choices AS pc
 				LEFT JOIN {db_prefix}log_polls AS lp ON (lp.id_choice = pc.id_choice AND lp.id_poll = {int:id_poll} AND lp.id_member = {int:current_member} AND lp.id_member != {int:not_guest})
-			WHERE pc.id_poll = {int:id_poll}',
+			WHERE pc.id_poll = {int:id_poll} ORDER by pc.id_choice',
```

Comment:
BY should be in capitalcase,
the ORDER BY should be in a new line

I'm not 100% sure if this sort shouldn't be done in php instead of db side.

Reply:
> BY should be in capitalcase,
> the ORDER BY should be in a new line

I'll fix that (sometime later this week).


> I'm not 100% sure if this sort shouldn't be done in php instead of db side.

I wouldn't know how to do that.

Reply:
Why would we do this sort in PHP when we can do it in SQL?

Reply:
In this case there isn't any advantage of using PHP versus doing it on the DB.